webpack.config.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  entry: './index.jsx',
+  output: {
+    path: './dist',
+    filename: 'bundle.js'
+  },
+  resolve: {
+    extensions: ['', '.jsx', '.js']
```

Comment:
If we resolve '.jsx' only, we will fail requiring 'react', as it's actually using '.js' resolution:

> In file node_modules/react/react.js

````js
require('./lib/React') // actually -> ./lib/React.js
````